LiteEditor/clang_driver.cpp

```diff
@@ -326,6 +326,11 @@ FileTypeCmpArgs_t ClangDriver::DoPrepareCompilationArgs(const wxString& projectN
         cppCompileArgs.Add(wxT("-std=c++14"));
         // cCompileArgs.Add(wxT("-std=c++14"));
     }
+	
+	if(workspaceFlags & LocalWorkspace::EnableCpp17) {
+        cppCompileArgs.Add(wxT("-std=c++17"));
+        // cCompileArgs.Add(wxT("-std=c++14"));

```

Comment:
You forgot to change "c++14" to "c++17" in this comment too :) I know it's only a comment, but leaving it as it is may be confusing in the future.